packaging/systemd/microshift-ovsdb-server.conf

```
@@ -1,2 +1,3 @@
 [Service]
 CPUAffinity=0
+ExecStartPre=-/bin/sh -c 'if [ -f /usr/lib/sysusers.d/openvswitch-hugetlbfs.conf ]; then usermod -a -G hugetlbfs openvswitch fi'
```

```suggestion
ExecStartPre=-/bin/sh -c 'if [ -f /usr/lib/sysusers.d/openvswitch-hugetlbfs.conf ]; then usermod -a -G hugetlbfs openvswitch; fi'
```

The command is missing a semicolon, so it's failing when the unit starts.